server/src/application/services/cardService.js

```diff
@@ -36,7 +31,7 @@ const cardService = {
 	checkCardExists: function (cardId) {
 		const card = cardRepository.getCardById(cardId);
 		if (!card) {
-			throw new Error("Card not found");
+			throw new Error("CARD_NOT_FOUND");
```

Comment:
On peut peut-être définir une bibliotèque avec les erreurs pour afficher le text lié à l'erreur CARD_NOT_FOUND
`errors = { CARD_NOT_FOUND: "Card not found", ... }`